src/cards/suffocate.ts

```diff
@@ -84,7 +84,7 @@ export function updateTooltip(unit: Unit.IUnit) {
     const turnsUntilSuffocation = Math.ceil(2 * Math.log2(unit.health / 10) + 1) - modifier.quantity;
 
     // Set tooltip:
-    modifier.tooltip = i18n(['turns until suffocation', turnsUntilSuffocation.toString()]);
+    modifier.tooltip = `${turnsUntilSuffocation} ${i18n('turns until suffocation')}`;
```

Comment:
This should've worked as is, if it didn't there may be something wrong with the i18n function.  It's supposed to replace the first bread emoji and there's one at the start of the localization string.

Reply:
Oh interesting. The format it's using now is similar to poison, but maybe poison is a weird exception because it has multiple variables.

Should we look at the i18n function?